src/commcare_cloud/commands/ansible/ops_tool.py

```diff
@@ -0,0 +1,122 @@
+            print(host_address + " not known_host " + self.known_host_file + ", or SSH failed"
+                  "\nRun `cchq <env> update-local-known-hosts`")
+            exit(1)
+        return output
+
+
+class ListDatabases(CommandBase):
+    command = 'list-postgresql-dbs'
+    help = """
+
+    Example:
+
+    To list all database on a particular environment.
+
+    ```
+    commcare-cloud <ev> list-databases
+    ```
+    """
+
+    arguments = (
+        Argument('--compare', action='store_true', help=(
+            "Gives additional databases on the server."
+        )),
+    )
+
+    def run(self, args, manage_args,compare=None):
+        # Initialize variables
+        dbs_expected_on_host = collections.defaultdict(list)  # Database that should be in host
+        dbs_present_in_host = collections.defaultdict(list)   # Database that are in host
+
+        args.server = 'postgresql'
+        ansible_username = 'ansible'
+        command = "sudo -iu postgres python /usr/local/sbin/db-tools.py  --list-all"
+
+        environment = get_environment(args.env_name)
+        ansible_password = environment.get_ansible_user_password()
+        host_addresses = get_instance_group(args.env_name, args.server)
+        known_host_file = environment.paths.known_hosts
+
+        if args.compare:
+            privileged_command = PrivilegedCommands(ansible_username, ansible_password, known_host_file, command)
+
+            # List from Postgresql query.
+            for host_address in host_addresses:
+                output = privileged_command.run(host_address)
+                for database in output:
+                    if database not in ['postgres', 'template0', 'template1']:
+                        dbs_present_in_host[host_address].append(database)
+
+        # List from Generated Mapping
+        dbs = environment.postgresql_config.to_generated_variables()['postgresql_dbs']['all']
+        for db in dbs:
+            dbs_expected_on_host[db['host']].append(db['name'])
+
+        # Print Logic
+        if args.compare:
+            # Printing Comparison
+            for host_address in dbs_present_in_host.keys():
+                print(host_address + ":")
+                print(" " * 4 + "Expected Databases:")
+                for database in dbs_expected_on_host[host_address]:
+                    print(" " * 8 + "- " + database)
+
+                print(" " * 4 + "Additional Databases:")
+                for database in dbs_present_in_host[host_address]:
+                    if database not in dbs_expected_on_host[host_address]:
+                        print(" " * 8 + "- " + database)
+        else:
+            # Print Default
+            for host_address in dbs_expected_on_host.keys():
+                print(host_address + ":")
+                print(" " * 4 + "Expected Databases:")
+                for database in dbs_expected_on_host[host_address]:
+                    print(" " * 8 + "- " + database)
+
```

Comment:
Quite a lot of duplication in the output logic. Could be clean up do something like this (also you can use `clint.textui` to do indenting and more:
```python
    dbs_expected_by_host = get_expected_dbs()
    dbs_present_by_host = None
    if args.compare:
        dbs_present_by_host = get_present_dbs()
    
    for host_address, dbs_expected in dbs_expected_by_host.items():
        dbs_present = dbs_present_by_host.get(host_address, {}) if dbs_present_by_host is not None else None
        puts(colored.blue(host_address + ":"))
        with indent():
            puts("Expected Databases:")
            with indent():
                for database in dbs_expected:
                    puts("- " + database)
                    dbs_present and dbs_present.pop(database, None)

            if dbs_present:
                puts(colored.orange("Additional Databases:"))
                with indent():
                    for database in dbs_present:
                        puts("- " + database)
```